hypothesis-python/tests/cover/test_reflection.py

```diff
@@ -758,11 +757,41 @@ def test_cache_key_size_is_bounded():
     f.__code__ = f.__code__.replace(
         co_consts=tuple(c * 1000 if c == "a" else c for c in f.__code__.co_consts)
     )
-    assert len(repr(reflection._lambda_source_key(f))) > 1000
-    assert len(repr(reflection._lambda_source_key(f, bounded_size=True))) < 1000
+    assert len(repr(lambda_sources._function_key(f))) > 1000
+    assert len(repr(lambda_sources._function_key(f, bounded_size=True))) < 1000
 
 
-def test_lambda_source_key_distinguishes_alpha_renames():
+def test_function_key_distinguishes_alpha_renames():
     # these terms are equivalent under the lambda calculus, but their
     # representations are not, so they should be cached differently.
-    assert _lambda_source_key(lambda x: x) != _lambda_source_key(lambda y: y)
+    assert lambda_sources._function_key(lambda x: x) != lambda_sources._function_key(
+        lambda y: y
+    )
+
+
+def test_import():
+    import time as t
+
+    f = lambda: t.ctime()
+    assert get_pretty_function_description(f) == "lambda: t.ctime()"
+
+
+@pytest.mark.parametrize("nop_on_f", [True, False])
+def test_code_normalization(nop_on_f):
+    f = lambda x: x
+    g = lambda x: x
+    h = f if nop_on_f else g
+    assert lambda_sources._function_key(f) == lambda_sources._function_key(g)
+
+    # Append a NOP to one of the bytecodes
+    h.__code__ = h.__code__.replace(co_code=h.__code__.co_code + b"\x09\x00")
+    assert lambda_sources._function_key(f) != lambda_sources._function_key(g)
```

Comment:
is `b"\x09\x00"` the hardcoded bytecode for `pass` or `...`? If so, is it easy to compute this at runtime, so it self-documents?

[e, later]: I see this is from `NOP = 9`. Pull those out to module-level constants, and import it in this test file?